templates/arista_eos_show_ip_bgp_summary_vrf.template

```diff
@@ -0,0 +1,13 @@
+Value Filldown VRF (\S+)
+Value Filldown ROUTER_ID (\d+?\.\d+?\.\d+?\.\d+?)
+Value Filldown LOCAL_AS (\d+)
+Value Required BGP_NEIGH (\d+?\.\d+?\.\d+?\.\d+?)
+Value NEIGH_AS (\d+)
```

Comment:
Should this be `REQUIRED` as well to match the **cisco_nxos** template?

Also, do you mind separating these out into two different PRs?

When you do that, can you change any static spacing within the state to either **\s*** or **\s+**? As well as add the following to the end of each state:
```
  ^\s*$$
  ^. -> Error
```

Once those are addressed and tests pass, I will get them merged!